### ISSUE TITLE
[vmap x vjp] Most used `nn.functional.<x>` ops: 1-25

### ISSUE THREAD

**Issue by zou3519** [facebookresearch/functorch#154]
- [ ] softmax
- [ ] relu
- [ ] interpolate
- [ ] pad
- [ ] log_softmax
- [ ] normalize
- [ ] cross_entropy
- [ ] grid_sample
- [ ] one_hot
- [ ] mse_loss
- [ ] conv2d
- [ ] dropout
- [ ] softplus
- [ ] sigmoid
- [ ] linear
- [ ] gelu
- [ ] avg_pool2d
- [ ] max_pool2d
- [ ] nll_loss
- [ ] embedding
- [ ] tanh
- [ ] leaky_relu
- [ ] adaptive_avg_pool2d
- [ ] cosine_similarity
- [ ] unfold
